packages/react-link/src/components/Link/Link.types.ts

```diff
@@ -122,6 +128,7 @@ export interface ILinkStyleProps {
   className?: string;
   isButton?: boolean;
   isDisabled?: boolean;
+  isUnderlined?: boolean;
```

Comment:
Wonder if this should be called just `underline` to match the prop name? But I could go either way on this since some of the other Link style props use the `is` convention.

Reply:
Yeah, went with the convention here but I can change that if wanted.